docs/faq.md

```diff
@@ -0,0 +1,80 @@
+
+## What's the difference between Flotiq and WordPress?
+
+WordPress at its core is traditional CMS with data entangled with the presentation,
+it introduced the headless version some time ago, but it still is not based on Open API, 
+so using it is more challenging than using Flotiq API.
+Learn more in our [blog post](https://flotiq.com/blog/why-you-should-consider-flotiq-headless-cms-over-wordpress).
+
+## What's the difference between Flotiq and other headless CMSes?
+
+Flotiq API is Open API compatible. In Flotiq, documentation grows with your system.
+At every stage, you can view and share the current API doc and 
+generate a code library tailored to your custom data model.
+Development teams can collaborate easily using automatically generated docs, 
+SDKs and API sandboxes based on the current data model.
+These solutions aim to improve the efficiency of development teams, 
+reduce costs by making the work parallel, and help to prototype.
+
+## How much does the Flotiq cost?
+
+Primary usage is free, and if that is not enough for your needs, you can check pricing 
+[here](https://flotiq.com#pricing){target="_blank"}.
+
+## What programming language I have to know to use Flotiq?
+
+You do not necessarily need to know any programming language to keep your data in Flotiq.
+You can use our starters to show your data to the world ([Gatsby](Universe/gatsby.md), [Scully](Universe/scully.md)).
+Knowing any of the front-end web programming language (JavaScript, Python, Ruby) will help you when you want to 
+fit the starters more to your needs or when our starters would not be close enough to your use case.
+
+## How to migrate existing service to Flotiq?
+
+The answer varies based on what service you have.
+We have out of the box migration for projects based on WordPress -
+[Move WordPress to Flotiq](https://flotiq.com/blog/how-to-move-from-wordpress-to-jamstack-using-our-gatsby-wordpress-starter).
+We are working on writing migrations tips and importers for other types of projects source.
+Please contact us to tell us what kind of service you wish to migrate to receive a more fitting solution.
+
+## Can I use Flotiq to make an online shop?
+
+Yes. U can use Flotiq to store your products, but we do not provide customer accounts or payment methods. 
+Our shop starters use [Snipcart](https://snipcart.com/) to manage purchase process.
+You can find shop starters [here](https://github.com/flotiq?q=gatsby-starter-products) 
+and [here](https://github.com/flotiq?q=scully-products).
+
+## Can I integrate Flotiq with other tools like Mailchimp or Gmail?
+
+We have [Zapier integration](https://zapier.com/apps/flotiq/integrations) ([docs](Universe/zapier.md)),
+so you can integrate thousands of tools with Flotiq fast and painless.
+
+## Can I recover my deleted data?
+
+If you need to bring back the deleted object, you can contact us on [hello@flotiq.com](mailto:hello@flotiq.com),
```

Comment:
I would change the answer for this question like this:

If you need to bring back the deleted object, you can contact us on [hello@flotiq.com](mailto:hello@flotiq.com).
Please include your email, content type name, and object id in the email.
It can be recovered from the backups, but remember that we keep them for a short time, which unfortunately vary,
only to bring the service back to work if something went wrong with deploying the new version or servers.

Reply:
suggestion applied